src/core/methods/initApp/websocket/websocket.constants.ts

```diff
@@ -0,0 +1,21 @@
+import { Socket } from 'socket.io-client';
+
+export type BatchTransactionsWSResponseType = {
+  batchId: string;
+  txHashes: string[];
+};
+
+export enum WebsocketConnectionStatusEnum {
+  NOT_INITIALIZED = 'not_initialized',
+  PENDING = 'pending',
+  COMPLETED = 'completed'
+}
+
+export const websocketConnection: {
```

Comment:
I'd rename `websocketConnection ` into `websoket` and `current` prop into `connection`
or
keep `websoketConnection` as name and rename `current` into `instance` or `channel`